vaadin-components-gwt/src/main/java/com/vaadin/components/grid/config/JSColumn.java

```diff
@@ -18,6 +18,18 @@
     void setName(String s);
 
     @JsProperty
+    String getHeaderText();
+
+    @JsProperty
+    void setHeaderText(String s);
+
+    @JsProperty
+    String getHidingToggleCaption();
```

Comment:
Alright. In that case I would perhaps name the attribute `hiding-toggle-text`, to align with `header-text`.